projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -241,6 +241,49 @@ public static function get_initial_state() {
 			/** This filter is documented in plugins/jetpack/modules/subscriptions/subscribe-module/class-jetpack-subscribe-module.php */
 			'isSubscriptionModalEnabled'  => apply_filters( 'jetpack_subscriptions_modal_enabled', false ),
 			'socialInitialState'          => self::get_publicize_initial_state(),
+			'gutenbergInitialState'       => self::get_gutenberg_initial_state(),
+		);
+	}
+
+	/**
+	 * Get information about the Gutenberg plugin and its Interactivity API support.
+	 *
+	 * @see https://make.wordpress.org/core/tag/interactivity-api/
+	 *
+	 * @return array
+	 */
+	private static function get_gutenberg_initial_state() {
+		$initial_state = array(
```

Comment:
This is only used once, it seems to me we may as well move it down to line 281. But feel free to ignore this suggestion if you disagree.

Reply:
👍 52e06639eb7687885e90de51a1ca585ba9f01fbe